Scenes/Creeps/Creep.gd

```diff
@@ -49,6 +49,7 @@ var _id: int
 @onready var _sprite: AnimatedSprite2D = $Visual/Sprite
 @onready var _health_bar = $Visual/HealthBar
 @onready var _selection_area: Area2D = $Visual/SelectionArea
+@onready var _creep_specials_container = $Visual/CreepSpecialsContainer
```

Comment:
Please rename to something else, like "SpecialIconsContainer".
"_creep_specials_container" sounds like something which would contain the specials themselves, rather than their visual representations.

Reply:
Renamed